README.md

```diff
@@ -92,19 +92,52 @@ The R53 domains for non-production and production are stored in different AWS ac
 
 example `extensions.yml` config.
 
+```yaml
+my-application-alb:
+  type: alb
+  environments:
+    dev: 
+      additional_address_list:
+        - internal.my-web-service-2
+```
+
+## CDN
+
+This module will create the CloudFront (CDN) endpoints for the application if enabled.
+
+Additional domains (`cdn_domains_list`) are the domain names that will be configured in CloudFront. In the map the key is the fully qualified domain name and the value is the application's base domain (the application's Route 53 zone).  
+
+example `extensions.yml` config.
+
 ```yaml
 my-application-alb:
   type: alb
   environments:
     dev: 
       cdn_domains_list:
-        dev.my-application.uktrade.digital: my-application.uktrade.digital
+        - dev.my-application.uktrade.digital: [ "internal", "my-application.uktrade.digital" ] 
+        - dev.my-web-service-2.my-application.uktrade.digital: [ "internal.my-web-service-2", "my-application.uktrade.digital" ]
       additional_address_list:
         - internal.my-web-service-2
+      enable_cdn_record: false
+      enable_logging: true
     prod:
-      domain: {my-application.great.gov.uk: "great.gov.uk"} 
+      cdn_domains_list: {my-application.great.gov.uk: "great.gov.uk"} 
```

Comment:
Is it a problem that this isn't a list?

Reply:
Its a list of key: values eg.
{my-application.great.gov.uk: "great.gov.uk", my-secondapp.great.gov.uk: "great.gov.uk"}

Reply:
@gabelton is that ok, or am I missunderstanding your comment?

Reply:
but it's technically a dict / map? Should it not be in the same format as `cdn_domains_list` under dev above ?

Reply:
oh my, you are correct.  I missed this, updaing.

Reply:
updated